js/blocks/ExtrasBlocks.js

```diff
@@ -886,6 +886,66 @@ function setupExtrasBlocks() {
         }
     }
 
+    class DisplayGridBlock extends FlowBlock {
+        constructor() {
+            super("displaygrid", _("Display Grid"));
```

Comment:
please use lowercase... "display grid" to be consistent with other block names.

Reply:
should I change Cursor events to lower case as well ? 
#2332

Reply:
Just the block labels need to be lower case.